test-cases/RMLTC0007h-CSV/mapping.ttl

```diff
@@ -15,7 +15,7 @@
 
   rr:subjectMap [ 
     rr:template "http://example.com/Student/{ID}/{FirstName}";
-    rr:graph [ rml:reference "Name"; rr:termType rr:Literal; ]
+    rr:rr:graphMap [ rml:reference "Name"; rr:termType rr:Literal; ]
```

Comment:
this is a typo, no? rr:rr:graphMap? Probably you have the same typo in the other mapping files (which might be the reason the mapper fails?)

Reply:
Actually it follows the R2RML spec: https://www.w3.org/TR/r2rml/#named-graphs

Reply:
Solve the typo, but it doesn't have any relation with the generation step, as the input is metadata.csv